### PR TITLE
Updated docs/overview.md and docs/vscode.md files.

### PR DESCRIPTION
**Pull request by kvignesh1420**
This PR fixes/updates the `docs/overview.md` and `docs/vscode.md` files.

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -6,12 +6,17 @@
...
-Prometheus, Apache Kafka, Apache Ignite, Google Cloud PubSub, AWS Kinesis,
-Microsoft Azure Storage, Alibaba Cloud OSS etc.
+TensorFlow I/O is an extension package to Tensorflow, which encompasses io support for 
+a collection of file systems and file formats that are not available in TensorFlow's built-in support.
+Integrations with many systems and cloud vendors include (but not limited to):
+
+- Prometheus
+- Apache Kafka
+- Apache Ignite
+- Google Cloud PubSub
```
Comment (vlasenkoalexey) [inline docs/overview.md:16]: Since you already updating this, would you mind adding Google Cloud BigQuery here?
It is used more widely than PubSub.

Comment (kvignesh1420) [reply docs/overview.md:17]: Added in the new commit.

**Comment by kvignesh1420** [comment]
@vlasenkoalexey , I added the item to the list.

**Pull request merged**